migrations/20210806091509_use-date.ts

```diff
@@ -0,0 +1,19 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('appointments', (table) => {
+    table.date('appointmentDate').alter();
+  });
+  return knex.schema.alterTable('psychologists', (table) => {
+    table.date('inactiveUntil').alter();
+  });
```

Comment:
* We could use `Promise.all` with the 2 requests.
* We could also alter `dateOfBirth` in `patients` table.